pytorch_lightning/plugins/native_amp.py

```diff
@@ -19,8 +19,7 @@ class NativeAMP:
     def __init__(self, trainer):
         self.trainer = trainer
 
-    def connect(self, model, optimizers):
-        self.trainer.optimizers = optimizers
+    def connect(self, model, optimizers=None):
```

Comment:
are there type annotations that can be added?

Reply:
not sure if it's required for internal apis.

Reply:
maybe not required but we shall add it everywhere at lias some basic (not all details)